compose/install/compose-plugin.md

```diff
@@ -0,0 +1,194 @@
+
+    > Or, if using a different distro, use the equivalent package manager instructions. 
+
+
+    ```console
+    $ sudo apt-get update
+    $ sudo apt-get install docker-compose-plugin
+    ```
+    
+    Alternatively, to install a specific version of Compose CLI plugin:
+      
+    a. List the versions available in your repo:
+
+
+      ```console
+      $ apt-cache madison docker-compose-plugin
+
+        docker-compose-plugin | 2.3.3~ubuntu-focal | https://download.docker.com/linux/ubuntu focal/stable arm64 Packages
+      ```
+
+    b. From the list obtained use the version string you can in the second column to specify the version you wish to install. 
+      
+    c. Install the selected version:
+
+
+      ```console
+      $ sudo apt-get install docker-compose-plugin=<VERSION_STRING>
+      ```
+    where `<VERSION_STRING>` is, for example,`2.3.3~ubuntu-focal`.
+
+3.  Verify that Docker Compose is installed correctly by checking the version.
+
+    ```console
+    $ docker compose version
+    Docker Compose version v2.3.3
+    ```
+
+### Install the plugin manually
+
+> **Note**
+>
+> These instructions assume you already have Docker Engine and Docker CLI installed and now want to install the Compose plugin. 
+>
+> Note as well this option requires you to manage upgrades manually. Whenever possible we recommend any of the other installation methods listed. For other Linux installation methods see [this summary](#installation-methods).
+
+>To run Compose as a non-root user, see [Manage Docker as a non-root user](../../engine/install/linux-postinstall.md).
+
+
+1.  To download and install the Compose CLI plugin, run:
+
+    ```console
+    $ DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+    $ mkdir -p $DOCKER_CONFIG/cli-plugins
+    $ curl -SL https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-linux-x86_64 -o $DOCKER_CONFIG/cli-plugins/docker-compose
+    ```
+
+    This command downloads the latest release of Docker Compose (from the Compose releases repository) and installs Compose for _the active user_ under `$HOME` directory. 
+    
+    > To install:
+    >* Docker Compose for _all users_ on your system, replace `~/.docker/ cli-plugins` with `/usr/local/lib/docker/cli-plugins`.
+    >* A different version of Compose, substitute `{{site.compose_version}}` with the version of Compose you want to use.
+
+2. Apply executable permissions to the binary:
+
+     ```console
+    $ chmod +x $DOCKER_CONFIG/cli-plugins/docker-compose
+    ```
+    or, if you chose to install Compose for all users:
+
+    ```console
+    $ sudo chmod +x /usr/local/lib/docker/cli-plugins/docker-compose
+    ```
+
+3. Test the installation.
+
+    ```console
+    $ docker compose version
+    Docker Compose version {{site.compose_version}}
+    ```
+
+> **Note**
+>
+>__Compose standalone__: If you need to use Compose without installing the Docker CLI, the instructions for the standalone scenario are similar. 
```

Comment:
Should this be a new heading as we are providing instructions on how to install Compose as a standalone?